components/nexusoperations/workflow/commands_test.go

```diff
@@ -404,7 +417,7 @@ func TestHandleCancelCommand(t *testing.T) {
 		require.Equal(t, 1, len(tcx.history.Events)) // Only scheduled event should be recorded.
 	})
 
-	t.Run("sets event attributes and spawns cancelation child machine", func(t *testing.T) {
+	t.Run("sets event attributes with UserMetadata and spawns cancelation child machine", func(t *testing.T) {
```

Comment:
Cancel shouldn't really have metadata IMHO, it's not something that we expose in the SDKs directly. Cancelation is propagated through the context or cancelation scopes.

Reply:
Yeah sure.